src/insider-threats/general-information.md

```diff
@@ -0,0 +1,70 @@
+3.  Read: [Google's North Korean Cyber Structure and Alignment](https://cloud.google.com/blog/topics/threat-intelligence/north-korea-cyber-structure-alignment-2023)
+4.  Read: [Google's advisory on DPRK IT Workers expanding scope and scale](https://cloud.google.com/blog/topics/threat-intelligence/dprk-it-workers-expanding-scope-scale)
+
+#### What are the goals of DPRK IT Workers?
+1.  Generating stable revenue for the North Korean regime through remote IT work.
+2.  Building support networks for North Korean IT-related operations (mules and money laundering).
+3.  Gaining access to Western companies technology, infrastructure, and identities (both personal and company, digital and physical).
+4.  Exfiltrating company secrets (intentionally and unintentionally).
+5.  Extortion (ransomware and blackmail).
+6.  Avoiding sanctions (North Korean entities are barred from receiving any form of payment from Western countries).
+7.  Hacking (establishing permanent access to infrastructure they are allowed into).
+8.  Malware (infecting high-value targets for future theft).
+9.  Read: [DPRK IT Workers pushing malicious code and stealing 1.3M](https://x.com/zachxbt/status/1824047425822310580)
+10. Read: [Highlights from the UN Security Council on DPRK IT Workers](https://x.com/tayvano_/status/1777911893224808911)
+
+#### How many DPRK IT Workers are there?
+
+The exact number is hard to estimate. DPRK IT Workers operate in separate 'cells,' and their organizational structure is flexible (even chaotic). Some workers focus exclusively on the blockchain space, while others are never seen applying outside of Fortune 500 companies. Estimates by different companies and government agencies vary between 2,000 and 15,000 in total. However, it's worth noting that this number includes multiple identities being re-used by the same actor or inactive accounts. **We estimate that about 3-5% of all Web3 developers are North Korean, and at any given time, there are at least 200-300 DPRK-related accounts actively seeking employment in Web3 companies.**
+
+#### Average DPRK IT Worker Profile
+
+In the following sections of this framework, we will discuss more advanced methods for detecting deeply nested DPRK IT Workers who may avoid the usual heuristics. However, it is still important to review the most common indicators of North Korean insider threats. If you suspect any of your contributors or have received a report regarding one, this set of questions can help you evaluate if you are dealing with a typical North Korean operative.
+
+1.  **Did you meet your contributor in real life?**
+    1.  Yes - Significantly lowers the chance that the contributor is North Korean.
+    2.  No - Red flag.
+2.  **Is your contributor an Asian man in his 20s or 30s?**
+    1.  Yes - Red flag.
+    2.  No - Lowers the chance that the contributor is North Korean, but they could still be a 'facilitator'.
```

Comment:
```suggestion
    2.  No - Lowers the chance that the contributor is North Korean, but they could still be a 'mule' (local accomplice or facilitator).
```
Added context and re-used the term 'mule' from earlier

Reply:
Decided to use 'facilitator' (changed earlier definition too). 'Mule' is more appropriate for money laundering related operatives, 'facilitator' role is broader than this. Facilitator can be a mule, but mule may not be a facilitator.